Gems/Atom/Tools/ShaderManagementConsole/Code/Source/Window/ShaderManagementConsoleTableView.cpp

```diff
@@ -25,10 +51,42 @@ namespace ShaderManagementConsole
         setEditTriggers(QAbstractItemView::NoEditTriggers);
         setSelectionBehavior(QAbstractItemView::SelectItems);
         setSelectionMode(QAbstractItemView::SingleSelection);
-        horizontalHeader()->setSectionResizeMode(QHeaderView::ResizeToContents);
+        verticalHeader()->setSectionResizeMode(QHeaderView::ResizeToContents);
+        horizontalHeader()->setSectionResizeMode(QHeaderView::Interactive);
+        setAlternatingRowColors(true);
 
         RebuildTable();
         AtomToolsFramework::AtomToolsDocumentNotificationBus::Handler::BusConnect(m_toolId);
+        this->setContextMenuPolicy(Qt::CustomContextMenu);
+        this->connect(this, &QTableWidget::customContextMenuRequested, this, &ShaderManagementConsoleTableView::ShowContextMenu);
+    }
+
+    void ShaderManagementConsoleTableView::mousePressEvent(QMouseEvent* e)
+    {
+        QTableWidget::mousePressEvent(e);
+        if (e->button() == Qt::RightButton)
+        {
+            ShowContextMenu(e->pos());
```

Comment:
Do we need this and the custom context menu requested connection?

Reply:
I think the connection is useless it doesn't work, thus this manual popup.